Library/Formula/spl.rb

```diff
@@ -0,0 +1,27 @@
+class Spl < Formula
+  homepage "http://www.clifford.at/spl/"
+  url "http://www.clifford.at/spl/releases/spl-1.0pre6.tar.gz"
```

Comment:
Sadly, even if development has ground to a sudden halt, pre-release may mean it ends up in ` devel-only `. I'll let one of the maintainers here make the decision though - I have no "power" over the core :wink: